spec/ttt/core_spec.clj

```diff
@@ -1,14 +1,59 @@
 (ns ttt.core-spec
-  (:require [speclj.core :refer :all]
-            [ttt.core :refer :all]))
+  (:require [speclj.core :refer [describe context it should= should-invoke stub with-stubs]]
+            [ttt.core :as game]
+            [ttt.game-rules :as rules]
+            [ttt.ui :as ui]
+            [ttt.board :as board]
+            [ttt.player :as player]))
 
+(def current-state {:board [0 1 2 3 4 5 6 7 8] 
+                    :players [{:marker "X"} {:marker "O"}] 
+                    :current-player {:marker "X"}})
 
+(def new-state {:board [0 1 2 3 "X" 5 6 7 8]
+                :players [{:marker "O"} {:marker "X"}]
+                :current-player {:marker "O"}})
 
-(describe "Game"
-  (context "#set-game-state"
-    (it "has a game state"
-      (should= {:board [0 1 2 3 4 5 6 7 8], :players [{:marker "X"} {:marker "O"}] :current-player {:marker "X"}}
-        (set-game-state [0 1 2 3 4 5 6 7 8] [{:marker "X"} {:marker "O"}])))))
+(def state-after-two-moves {:board [0 1 2 3 "X" 5 6 7 "O"]
+                            :players [{:marker "X"} {:marker "O"}]
+                            :current-player {:marker "X"}})
 
+(def players [{:marker "O"} {:marker "X"}])
 
 
+(describe "Game"          
+  (context "#play"
+    (with-stubs)
+    
+    (it "calls game over message when the game is over"
+      (with-redefs [rules/game-over? (stub :rules/game-over? {:return true})]
+        
```

Comment:
I think for me personally, the whitespace here makes it a bit confusing because I expected the `it` block to not have a whitespace in the middle of it. I do see why you'd do that though. Is there a guideline for this somewhere?